cobc/cconv.h

```diff
@@ -0,0 +1,53 @@
+/*
+   Copyright (C) 2022 Free Software Foundation, Inc.  Written by Keisuke
+   Nishida, Roger While, Simon Sobisch, Edwart Hart, Ron Norman, Nicolas
+   Berthier
+
+   This file is part of GnuCOBOL.
+
+   The GnuCOBOL compiler is free software: you can redistribute it
+   and/or modify it under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of the
+   License, or (at your option) any later version.
+
+   GnuCOBOL is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with GnuCOBOL.  If not, see <https://www.gnu.org/licenses/>.
+*/
+/* CHECKME: I think adding this particular header file makes sense, but this
+   could also move to cobc.h for consistency with all other declarations. */
+#ifndef CB_CCONV_H
+#define CB_CCONV_H
+
+#include "config.h"
+
+#include <stdio.h>	  /* CHECKME: needed for definitions in cobc.h; could be
```

Comment:
this include and the ones above ad below should be replaced by `include "libcob/common.h"`.